nengo/tests/test_cache.py

```diff
@@ -524,12 +525,15 @@ def test_writeablecacheindex_removes(tmpdir):
 
 
 def test_writeablecacheindex_warning(monkeypatch, tmpdir):
-    def raise_error(*args, **kwargs):
-        raise CalledProcessError(-1, "move")
 
-    monkeypatch.setattr(os, "replace", raise_error)
     with pytest.warns(CacheIOWarning):
-        with WriteableCacheIndex(cache_dir=str(tmpdir)):
+        with WriteableCacheIndex(cache_dir=str(tmpdir)) as idx:
```

Comment:
Yeah I didn't bother updating some of the non-core test files (e.g. in utils and here) if it would have involved a lot of lines of code, just didn't seem worth the effort.